### PR TITLE
[MiniAOD, NanoAOD] Update AK8 subjet b-taggers, add pileup jet ID 

### PR DESCRIPTION
**Pull request by nurfikri89**
#### PR description:

This PR adds the following in MiniAOD and NanoAOD:

- AK8 subjet taggers: Remove DeepCSV and older taggers. Add DeepJet (fallback option) and UParT (latest and greatest).
- Pileup Jet ID:  Following up PR #46137, now adds Pileup Jet ID discriminant in NanoAOD.

#### PR validation:

- passes the usual runTheMatrix test: `runTheMatrix.py -l limited -i all --ibeos`
- passes MiniAOD workflows: `runTheMatrix.py -i all --ibeos  -l 2500.021,2500.022,2500.023,2500.024,2500.031,2500.032,2500.033,2500.034`

### CODE REVIEW

**Comment by jfernan2** [comment]
enable profiling

**Comment by jfernan2** [comment]
please test

**Comment by hqucms** [comment]
enable nano,profiling

**Comment by hqucms** [comment]
please test

File: PhysicsTools/NanoAOD/python/nanoDQM_cff.py [hunk 1]
```diff
@@ -234,6 +238,15 @@ def _match(name):
     if 'area' not in plot.name.value():
         _SubJet_EarlyRun3_plots.append(plot)
 
+_SubJet_pre142X_plots = cms.VPSet()
+for plot in nanoDQM.vplots.SubJet.plots:
+    if 'btagDeepFlavB' not in plot.name.value() and 'btagUParTAK4B' not in plot.name.value():
+        _SubJet_pre142X_plots.append(plot)
+_SubJet_pre142X_plots.extend([
+    Plot1D('btagCMVA', 'btagCMVA', 20, -1, 1, 'CMVA V2 btag discriminator'),
```
Comment (hqucms) [inline PhysicsTools/NanoAOD/python/nanoDQM_cff.py:246]: Do we have `btagCMVA` for subjets?

File: PhysicsTools/NanoAOD/python/nanoDQM_cfi.py [hunk 2]
```diff
@@ -817,6 +818,8 @@
                 Count1D('_size', 9, -0.5, 8.5, 'slimmedJetsAK8PFPuppiSoftDropPacked::SubJets, i.e. soft-drop subjets for ak8 fat jets for boosted'),
                 Plot1D('btagCMVA', 'btagCMVA', 20, -1, 1, 'CMVA V2 btag discriminator'),
```
Comment (hqucms) [inline PhysicsTools/NanoAOD/python/nanoDQM_cfi.py:819]: I think `btagCMVA` can be removed.

File: PhysicsTools/NanoAOD/python/nanoDQM_cff.py (hunk 1 above)
Comment (nurfikri89) [reply PhysicsTools/NanoAOD/python/nanoDQM_cff.py:246]: Not anymore. Will remove this.

File: PhysicsTools/NanoAOD/python/nanoDQM_cfi.py (hunk 2 above)
Comment (nurfikri89) [reply PhysicsTools/NanoAOD/python/nanoDQM_cfi.py:819]: I will remove this also.

**Comment by hqucms** [comment]
please abort

**Comment by hqucms** [comment]
please test

**Comment by hqucms** [comment]
+1

**Comment by hqucms** [comment]
+1

**Comment by jfernan2** [comment]
+1

**Comment by mandrenguyen** [comment]
+1

**Pull request merged**